scripts/pre_commit_linter.py

```diff
@@ -78,6 +78,10 @@
     nargs='+',
     help='specific files to be linted. Space separated list',
     action='store')
+_PARSER.add_argument(
```

Comment:
I would suggest we have only one flag `-v` for verbose. There is no need to introduce the other flag -- it should be the default behavior.

Reply:
Okay.

Reply:
DONE .